Scripts/RemediationScripts/Remediate-DisableLocalAccountsForKubernetesService.ps1

```diff
@@ -0,0 +1,867 @@
+                    Write-Host $([Constants]::SingleDashLine)
+                }
+            }
+            else
+            {
+                $kubernetesClusterSkipped += $kubernetesServiceResource
+                $logResource = @{}
+                $logResource.Add("ResourceGroupName",($_.ResourceGroupName))
+                $logResource.Add("ResourceName",($_.Name))
+                $logResource.Add("Reason","local accounts can't be disabled for this resource since RBAC is disabled.")    
+                $logSkippedResources += $logResource
+                Write-Host "Local accounts can't be disabled for this resource since RBAC is disabled." -ForegroundColor $([Constants]::MessageType.Error)
+                Write-Host $([Constants]::SingleDashLine)
+                return
+            }
+        }
+       
+        $colsProperty = @{Expression={$_.Name};Label="Resource Name";Width=30;Alignment="left"},
+                        @{Expression={$_.ResourceGroupName};Label="Resource Group Name";Width=30;Alignment="left"},
+                        @{Expression={$_.Location};Label="Location";Width=20;Alignment="left"},
+                        @{Expression={$_.IsLocalAccountsDisabled};Label="Is local accounts disabled?";Width=30;Alignment="left"},
+                        @{Expression={$_.Id};Label="Resource ID";Width=80;Alignment="left"}
+
+        if($AutoRemediation)
+        {
+            if ($($kubernetesClusterRemediated | Measure-Object).Count -gt 0)
+            {
+                # Write this to a file.
+                $kubernetesClusterRemediatedFile = "$($backupFolderPath)\RemediatedKubernetesClusters.csv"
+                $kubernetesClusterRemediated | Export-CSV -Path $kubernetesClusterRemediatedFile -NoTypeInformation
+                Write-Host "Local accounts is enabled on the Kubernetes cluster(s)." -ForegroundColor $([Constants]::MessageType.Update)
+                Write-Host "`nThis information has been saved to [$($kubernetesClusterRemediatedFile)]." -ForegroundColor $([Constants]::MessageType.Update)
+                Write-Host $([Constants]::SingleDashLine)
+            }
+
+            if ($($kubernetesClusterSkipped | Measure-Object).Count -gt 0)
+            {
+                # Write this to a file.
+                $kubernetesClusterSkippedFile = "$($backupFolderPath)\SkippedKubernetesClusters.csv"
+                $kubernetesClusterSkipped | Export-CSV -Path $kubernetesClusterSkippedFile -NoTypeInformation
+                Write-Host "Error disabling local accounts on some Kubernetes cluster(s)." -ForegroundColor $([Constants]::MessageType.Error)
+                Write-Host "`nThis information has been saved to [$($kubernetesClusterSkippedFile)]." -ForegroundColor $([Constants]::MessageType.Update)
+                Write-Host $([Constants]::SingleDashLine)
+            }
+
+        }
+        else 
+        { 
+            Write-Host $([Constants]::DoubleDashLine)
```

Comment:
This double dash line should be outside this block and above if statement na?

Reply:
Okay. Updated.